pkgs/halo2-eth-membership-worker/package.json

```diff
@@ -27,8 +27,8 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@anonklub/halo2-eth-membership": "file:../halo2-eth-membership",
```

Comment:
#499 is patching `@anonklub/halo2-eth-membership`, so latest version will be `0.1.1`.
#499 must merge first.

Reply:
TODO: bump lockfile